### PR TITLE
Added initial documentation files

### PR DESCRIPTION
**Pull request by jsaarimaa**
Added initial documentation files README.md and CONTRIBUTING.md. More content will be added to these files when more features are impelemted.

### CODE REVIEW

**Review by sakari-malkki — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Px.Utils
+
+Px.Utils is a .NET library for reading and processing px files and processing data in px-type cube format. The goal of this library is to offer a high performance and easy to use library that can be integrated into any .NET project. This project aims to follow these core design principles:
+
+### High performance
+Everything is designed with performance in mind. Px files are commonly used in webserver environments where performance and low memory usage is important.
+### Be as unopinionated as possible
+We do not want to limit the user if not abolutely necessary. There are variations in the metadata between users and we want to support as many as possible.
```
Comment (LauraRocks) [inline docs/README.md:8]: Typo abolutely

**Review by LauraRocks — CHANGES_REQUESTED** [review]
Fix typo as indicated in the comment

File: docs/README.md (hunk 1 above)
Comment (jsaarimaa) [reply docs/README.md]: Fixed

**Review by sakari-malkki — APPROVED** [review]
Typo fixed, looks good

**Review by LauraRocks — APPROVED** [review]

**Pull request merged**